pkgs/shells/bash/default.nix

```diff
@@ -113,6 +113,6 @@ stdenv.mkDerivation rec {
   };
 
   passthru = {
-    shellPath = "/bin/bash";
+    shellPath = "${bash}/bin/bash";
```

Comment:
@markus1189 all packages that can be used as a login shell have to specify a stub shell path, I believe. So this should almost certainly be left alone.